ui/app/templates/components/transit-key-action/verify.hbs

```diff
@@ -222,5 +205,13 @@
         </button>
       </div>
     </div>
-  {{/if}}
 </form>
+<Modal @title="Results" @onClose={{action (mut isModalActive) false}} @isActive={{isModalActive}}>
+  <section class="modal-card-body">
+    <AlertBanner
+      @type={{if valid 'success' 'danger'}}
+      @title={{if valid 'Valid' 'Not Valid'}}
+      @message="The input is {{if valid 'valid' 'not valid'}} for the given {{if signature 'signature' 'hmac'}}"
```

Comment:
there it is.  that lowercase hmac.  Should be HMAC

Reply:
nice catch. fixed!